open-banking-accelerator/components/com.wso2.openbanking.accelerator.data.publisher/com.wso2.openbanking.accelerator.data.publisher.common/src/main/java/com/wso2/openbanking/accelerator/data/publisher/common/util/OBDataPublisherUtil.java

```diff
@@ -60,6 +63,16 @@ public static void releaseDataPublishingInstance(OpenBankingDataPublisher instan
      */
     public static void publishData(String streamName, String streamVersion, Map<String, Object> analyticsData) {
 
+        // Analytics data will be added to the OB analytics logfile for processing if ELK is configured for the server.
+        if (Boolean.parseBoolean((String) OpenBankingConfigParser.getInstance().getConfiguration()
+                .get(DataPublishingConstants.ELK_ANALYTICS_ENABLED))) {
+            try {
+                AnalyticsLogsUtils.addAnalyticsLogs("OB_LOG", streamName, streamVersion, analyticsData);
```

Comment:
Shall we OB_LOG to a constant file

Reply:
Fixed in 7e9a89d5aa64e24a84ef4f76b15ff07a393d5f5f